scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/UploadServlet.java

```diff
@@ -25,11 +25,14 @@
  */
 package org.ow2.proactive_grid_cloud_portal.scheduler.server;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)